examples/source-to-knative-service/app-operator/supply-chain-templates.yaml

```diff
@@ -61,7 +61,7 @@ kind: ClusterSourceTemplate
 metadata:
   name: test
 spec:
-  urlPath: .spec.inputs.source.url
+  urlPath: .status.outputs.url
```

Comment:
might be worth adding to the description that this will close https://github.com/vmware-tanzu/cartographer/issues/374